src/bytewise.rs

```diff
@@ -11,6 +11,9 @@ use alloc::vec::Vec;
 use crate::build_helper::BuildHelper;
 use crate::errors::{DaachorseError, Result};
 use crate::intpack::{U24nU8, U24};
+use crate::serializer::{
+    deserialize_vec, serialize_slice, serialized_bytes, Deserialize, Serialize,
```

Comment:
Could you follow the idiomatic way to bring fuctions?
https://doc.rust-lang.org/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#creating-idiomatic-use-paths
> Specifying the parent module when calling the function makes it clear that the function isn’t locally defined while still minimizing repetition of the full path.